client/cluster.go

```diff
@@ -411,10 +413,25 @@ func (c *ClusterClient) DrainNode(name string) error {
 func (c *ClusterClient) NodeExists(name string) (bool, error) {
 	c.logger.Info("client_node_exists", zap.String("name", name))
 
-	if err := c.kubectl.NodeExists(name); err != nil {
+	addPrefix := true
+
+	if c.cluster.CloudProvider() == api.AWS {
+		myvar, err := c.Machine(name)
+		if err != nil {
+			return false, err
+		}
+
+		myPrivateIP := strings.Replace(myvar.PrivateIP, ".", "-", -1)
```

Comment:
Prefixing with "my" feels a bit odd.
```suggestion
		privateIPString := strings.Replace(myvar.PrivateIP, ".", "-", -1)
```